packages/mobile/src/components/countdown/helpers/interval-setter.ts

```diff
@@ -0,0 +1,27 @@
+import { getStrictTimeToEvent } from '~/helpers/date/date';
+import { UPDATE_END_DATE_INTERVAL } from '~/common/constants/constants';
+
+type Time = {
+  days: string;
+  hours: string;
+  minutes: string;
+  seconds: string;
+};
+
+const intervalSetter = (
```

Comment:
I'd suggest converting it to a custom hook

Reply:
refactored